src/app/shared/services/shared.service.ts

```diff
@@ -179,7 +179,26 @@ export class SharedService {
    */
      public getOrdenesPagoById(id: string) {
       this.rqManager.setPath('CENTRAL_CUENTAS_CRUD_SERVICE');
-      return this.rqManager.getv2(`orden-pago/${id}`, null, null, null, null, null, 0);
+      return this.rqManager.getv2(`orden-pago/${id}`);
+    }
+
+     /**
+   * Gets OrdenesPago by doc
+   * @param id id de la orden de pago
+   * @returns  Documento
+   */
+      public getOrdenesPagoByDoc(doc: string) {
+        this.rqManager.setPath('CENTRAL_CUENTAS_CRUD_SERVICE');
+        return this.rqManager.getv2(`orden-pago/?query=DocumentoBeneficiario:${doc},Estado:Firmado`);
```

Comment:
Validar si funciona igual
```suggestion
        return this.rqManager.getv2(`orden-pago?query=DocumentoBeneficiario:${doc},Estado:Firmado`);
```